src/main/java/RepositoryCloner.java

```diff
@@ -0,0 +1,46 @@
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+
+public class RepositoryCloner {
+    
+    private static final String tempDirectoryPath = "clonedRepo";
+
+    static String cloneRepo(String repoHttpsAdress , String branchName, String repoName) throws IOException, GitAPIException {
+
+        try {
+            File tempDirectory  = new File(repoName);
+            boolean okCloning = true;
+
+            Git git = Git.cloneRepository().
+                    setURI(repoHttpsAdress).
+                    setBranch(branchName).
+                    setDirectory(new File(tempDirectoryPath)).
+                    call();
+
+            System.out.println((okCloning ? "Successful " : "Unsuccessful ") + "cloning of repo: " + repoHttpsAdress);
```

Comment:
```suggestion
            System.out.println("Successful cloning of repo: " + repoHttpsAdress);
```

Reply:
I wrote the previous code, with the new structure it's a bit redundant. So we can rewrite it without the one line if-statment